homestar-invocation/src/task/instruction/nonce.rs

```diff
@@ -88,6 +93,30 @@ impl TryFrom<&Ipld> for Nonce {
     }
 }
 
+impl JsonSchema for Nonce {
+    fn schema_name() -> String {
+        "nonce".to_owned()
+    }
+
+    fn schema_id() -> Cow<'static, str> {
+        Cow::Borrowed("homestar-invocation::task::instruction::Nonce")
+    }
+
+    fn json_schema(_gen: &mut SchemaGenerator) -> Schema {
+        let schema = SchemaObject {
+            instance_type: Some(SingleOrVec::Single(InstanceType::String.into())),
+            metadata: Some(Box::new(Metadata {
+                description: Some(
+                    "A 12-byte or 16-byte nonce. Use empty string for no nonce.".to_string(),
```

Comment:
minor: so period here right should not be a period?

Reply:
This one is a bit of an outlier because it contains two sentences. What would you think about following the bullet point convention in writing where you don't use a period for a single sentence, but you do when you have more than one?

Otherwise, I could rewrite this to a single sentence, but in general, that may artificially force us into always using a single sentence.